homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -194,8 +203,26 @@ def device_state_attributes(self):
                     STATE_ON if self.vacuum_state.dnd else STATE_OFF,
                 # Not working --> 'Cleaning mode':
                 #    STATE_ON if self.vacuum_state.in_cleaning else STATE_OFF,
-                ATTR_CLEANING_TIME: str(self.vacuum_state.clean_time),
-                ATTR_CLEANED_AREA: round(self.vacuum_state.clean_area, 2)})
+                ATTR_CLEANING_TIME: str(int(
+                    self.vacuum_state.clean_time.total_seconds()
+                    / 60)) + ' min',
```

Comment:
I'd suggest not adding units to the attributes. Values without units are easier to parse in a template for the users.

Reply:
Is there any other method to declare the units? The problem that I see is that the user doesn't know which unit it is. You have minutes, hours and square meters. The biggest problem would be to distinguish hours and minutes. Or I just mean that is a problem?

Reply:
At minimum we should make the units clear in the docs. I don't think there's another way to show it in the GUI. I'm not sure what's worse, and I don't think we have a rule for this yet in home assistant, but I would still favor my suggestion . 3rd person, or more, input is welcome.

Reply:
I agree with you. It is easier to work with the attribute values without units. Maybe this could be a new feature to add units to attributes to show them in the gui.